configs/katello/4.1.yaml

```diff
@@ -0,0 +1,111 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.1.0:
+    :redmine_version_id: 1351
+:prior_releases:
```

Comment:
Prior releases means that "any issues resolved in this prior release need to be included in 4.1". 

I think the accurate list of priors for 4.1 as of right now is:
- 4.0.1
- 3.18.4
- 3.18.3

..because all of those releases are likely to have changes merged in *after* 4.1 is branched or released which need to make it in to 4.1.x eventually otherwise we've basically regressed due to absence of those fixes. indicating these ensures that the cherry-picker looks in redmine to make sure everything is present in the KATELLO-4.1 release branch which you'll be creating soon.

Reply:
@jturel thanks, updated!  I am also keeping in 4.0.0 as per @jjeffers.

Reply:
There's no harm in it but I don't think it's needed since that code is finalized

Reply:
Since 4.0 is GA it can absolutely be removed from this list of priors as all of the code for 4.0 is merged by the time 4.1 branches